web/scripts/users/usersRepo.js

```diff
@@ -21,6 +21,20 @@ module.exports = {
       }, callback)
     }
 
-    return { login, register }
+    const addCat = (email, categories, callback) => {
+      repo.put({
+        path: '/users/' + email + '/addcat',
```

Comment:
As I mentioned on the API side, I would suggest making the same changes here with the URI.

Reply:
👍